conjure_oxide/tests/rewrite_tests.rs

```diff
@@ -914,6 +916,154 @@ fn rewrite_solve_xyz() {
     solver.solve(Box::new(|_| true)).unwrap();
 }
 
+#[test]
+fn rewrite_solve_xyz_parameterized() {
+    println!("Rules: {:?}", get_rules());
+
+    let rule_sets = match resolve_rule_sets(SolverFamily::Minion, &vec!["Constant"]) {
```

Comment:
I think line needs a '.to_string()', it broke building from a fresh checkout?

Reply:
I am quite surprised that something isn't building on a fresh checkout. I took those lines from the previous rewrite tests which did not seem to break anything and the tests passed all the CI build checks. I have also built this on a lab machine and it seems to build fine. I have only tried manually building on Ubuntu and Fedora though - what OS is this breaking on @ChrisJefferson?

Reply:
also fails in CI: https://github.com/conjure-cp/conjure-oxide/actions

Reply:
may be better to look at the recent commits actually: https://github.com/conjure-cp/conjure-oxide/commits/main/

Reply:
This is just an occasional issue with github, you can have two PRs (this one, and the simple CLI one) which both pass tests independantly, but don't merge together.

Basically github tests that (a) your branch is mergable, and (b) it passes all tests, but it (annoyingly) doesn't test that your branch will pass all tests once merged! I've always been suprised github doesn't have an option to fix this, but all solutions to this seem to be more work than it occasionally happening...